### PR TITLE
[ci skip] Update contributing guide

### PR DESCRIPTION
**Pull request by wazery**
Just mention the correct place for the new entries to the CHANGELOG.

### CODE REVIEW

**Comment by robin850** [comment]
Hi @wazery,

Thanks for the patch. Actually the fact that new entries should be at the top is already mentioned in the lines under the one you edited. I think @matthewd meant that we should highlight it more. Maybe we should remove the ["and it should got on top of a CHANGELOG"](https://github.com/rails/rails/pull/19732/files#diff-f7b1cb4f5ad678ab74407666d92a5502R360) part of the sentence and add a whole new line after "you should attach the issue's number" like:

> Make sure that new CHANGELOG entries get **at the top** of the file.

What do you think Matthew ?

**Comment by wazery** [comment]
Hi @robin850 

+1 for a new line and emphasizing the "at the top" part of the sentence.

I can amend the commit after Matthew's opinion.

**Comment by matthewd** [comment]
Hmm. I suspect this change would help as written, and we could just remove the now-redundant mention in the second paragraph. I actually looked for an "at the top" here last time I fixed one manually, and didn't see that it was already stated. :disappointed_relieved:

**Comment by wazery** [comment]
@matthewd I will remove the second mention and emphasize the "at the top" part.

**Comment by wazery** [comment]
Edited the commit, I hope it can be merged now.

File: guides/source/contributing_to_ruby_on_rails.md [hunk 1]
```diff
@@ -355,9 +355,9 @@ $ RUBYOPT=-W0 bundle exec rake test
...
 
-You should add an entry to the CHANGELOG of the framework that you modified if you're adding or removing a feature, committing a bug fix or adding deprecation notices. Refactorings and documentation changes generally should not go to the CHANGELOG.
+You should add an entry **to the top** of the CHANGELOG of the framework that you modified if you're adding or removing a feature, committing a bug fix or adding deprecation notices. Refactorings and documentation changes generally should not go to the CHANGELOG.
 
-A CHANGELOG entry should summarize what was changed and should end with author's name and it should go on top of a CHANGELOG. You can use multiple lines if you need more space and you can attach code examples indented with 4 spaces. If a change is related to a specific issue, you should attach the issue's number. Here is an example CHANGELOG entry:
+A CHANGELOG entry should summarize what was changed and should end with author's name. You can use multiple lines if you need more space and you can attach code examples indented with 4 spaces. If a change is related to a specific issue, you should attach the issue's number. Here is an example CHANGELOG entry:
```
Comment (kaspth) [inline guides/source/contributing_to_ruby_on_rails.md]: Let's fix this sentence too. There's a "the" missing in front of "author's".

Comment (wazery) [inline guides/source/contributing_to_ruby_on_rails.md]: Done.

**Comment by kaspth** [comment]
@wazery Thanks :wink:

**Comment by wazery** [comment]
@kaspth :smiley:

**Pull request merged**